featurex/stimuli/video.py

```diff
@@ -72,57 +72,13 @@ class DerivedVideoStim(VideoStim):
     VideoStim containing keyframes (for API calls). Each keyframe is associated
     with a duration reflecting the length of its "scene."
     """
-    def __init__(self, filename, **kwargs):
+    def __init__(self, filename, elements, frame_index, history):
```

Comment:
We may want to allow `frame_index=None` and `history=None` here. Granted, a `DerivedVideoStim` created by a `FrameSamplingConverter` will always have a known `history`, but I can imagine a use case where someone might just want a way to indicate that something is derived from a `VideoStim` even if they aren't passing `frame_index` and `history` properly. Since we're currently not using those attributes for anything else, it seems safe to allow `None` values. If at some point we extend the API for this class to the point where `history` is required, it'll make sense to mandate those arguments.